examples/06-custom-effects/README.md

```diff
@@ -1,24 +1,74 @@
+![](./header.png)
+
 # 06 Custom Effects
 
-We haven't quite got round to documenting this example yet. If you know a little
-bit about Lustre or Elm and want to help out, we'd love to have your help! Please
-[open an issue](https://github.com/lustre-labs/lustre/issues/new) if you have any
-ideas or reach out to @hayleigh.dev on the [Gleam discord](https://discord.gg/Fm8Pwmy).
+In the last example, we showed how to use effects provided by `lustre_http`. In
+this example we'll see how to create effects of our own using Lustre's
+[`effect.from`](https://hexdocs.pm/lustre/4.0.0-rc1/lustre/effect.html#from)
+method.
+
+Since we use Lustre effects to communicate with _external_ systems (like the
```

Comment:
I think we should consistently refer to this as just "effects" or `Effect` (singular) – I think people know we're talking about Lustre's stuff at this point ^.^